melodies_monet/_cli.py

```diff
@@ -1409,7 +1447,25 @@ def get_openaq(
             for vn in ["city", "is_analysis"]:  # may have been dropped for being all null
                 if vn not in df.columns:
                     site_vns.remove(vn)
-
+        elif method == "api-v3":
+            df = df.drop(
+                columns=[
+                    "sensor_id",
+                    "period_label",
+                ],
+                errors="ignore",
+            )
+            site_vns = [
+                "siteid",  # real OpenAQ location ID
+                "latitude",
+                "longitude",
+                "utcoffset",
+                #
+                "country",
+                #
+                "is_mobile",
+                "is_monitor",
```

Comment:
In the previous version, this was encompassed in a single string variable, `sensor_type`. We could keep that behavior here. That's probably the only thing that might change in this PR. Other potential changes, like for the rate limiting, would be just on the monetio side.

Reply:
I think it's okay to just leave the labels to be consistent with OpenAQ version 3 here and we do not have to keep the old behavior.